### DISCUSSION TITLE
Changing color of the single select [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2143] [Discord Threads]
> 🙋‍♂️ **Guineu**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### we are working on it and not possible at the moment  
#  
> 🙋‍♂️ **Guineu**  
#### ok! thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973188788937453598)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
